[nixos, nix] init with the current releases

06kellyjac opened the pull request:
Added EOL details for NixOS

(Draft so it can be reviewed by other NixOS people)

Synthetica9 commented on products/nixos.md:
```diff
@@ -0,0 +1,91 @@
+permalink: /nixos
+alternate_urls:
+  - /nixoslinux
+  - /nix
```
Maybe we should keep this one free for Nix itself?

06kellyjac replied:
Yeah good point

06kellyjac committed:
[nixos] init with the current releases

current and planned releases:
- `22.05`
- `21.11`
- `21.05`
- `20.09`
- `20.03`
- `19.09`
- `19.03`
- `18.09`
- `18.03`
- `17.09`
- `17.03`
- `16.09`
- `16.03`
- `15.09`
- `14.12`
- `14.04`
- `13.10`

mweinelt commented on products/nixos.md:
```diff
@@ -0,0 +1,92 @@
+
+Two main branches are offered: current Stable release and Unstable following latest development.
+
+**If you're looking for nix, the language and tool NixOS is based on click [here](./nix)**
```
```suggestion
**If you're looking for nix, the package manager NixOS is based on click [here](./nix)**
```

06kellyjac committed:
[nix] init with the current and most previous releases

release cyles:
- 2.8
- 2.7
- 2.6
- 2.5
- 2.4
- 2.3
- 2.2
- 2.1
- 2.0
- 1
